packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js

```diff
@@ -22,23 +22,21 @@ const ComponentSettingsView = () => {
   useEffect(() => {
     const CancelToken = axios.CancelToken;
     const source = CancelToken.source();
-
+  
```

Comment:
Whitespace that shouldn't be there. Why didn't the linter catch this? 🤔

Reply:
done